The linker on Darwin/x86 needs archives with an index, so create one.

--- Makefile.PL
+++ Makefile.PL
@@ -260,7 +260,7 @@
 	cover
 
 netdns$(LIB_EXT): netdns$(OBJ_EXT)
-	$(AR) '.($^O eq 'MSWin32' && $Config{'cc'} eq 'cl'?'/OUT:':'cr ').
+	$(AR) '.($^O eq 'MSWin32' && $Config{'cc'} eq 'cl'?'/OUT:':'crs ').
 	  'netdns$(LIB_EXT) netdns$(OBJ_EXT)
 
 ';
